receiver/redisreceiver/proto.go

```diff
@@ -46,7 +50,7 @@ func buildKeyspaceKeysMetric(k *keyspace, t *timeBundle) *metricspb.Metric {
 	m := &redisMetric{
 		name:   "redis/db/keys",
 		labels: map[string]string{"db": k.db},
-		mdType: metricspb.MetricDescriptor_CUMULATIVE_INT64,
+		mdType: metricspb.MetricDescriptor_GAUGE_INT64,
```

Comment:
@pmcollins can you add description for each metric? It is hard to know when one is gauge or cumulative.